pkg/generator/certificate.go

```diff
@@ -166,10 +166,49 @@ func GenerateSignedCert(rootCA *Certificate, algorithm v1alpha1.Algorithm, commo
 	if err != nil {
 		return cert, errors.WithStack(err)
 	}
-
 	return cert, nil
 }
 
+// GenerateSelfSignedCertPEM generates a self signed certificate with an expire date based on a unix timestamp
+func GenerateSelfSignedCertPEM(commonName string, expire int) ([]byte, []byte, []byte, error) {
```

Comment:
What is the difference between this function and `GenerateRootCA` besides PEM encoding everything? The root CA that `GenerateRootCA` creates is a self-signed cert. Maybe instead of reimplementing generation, this function could just call `GenerateRootCA` then PEM encode the things?

Reply:
And maybe we rename the function to `GenerateRootCAPEM`?